gst/nnstreamer/elements/gsttensor_transform.h

```diff
@@ -165,6 +165,7 @@ struct _GstTensorTransform
   GstTensorsConfig in_config; /**< input tensors config */
   GstTensorsConfig out_config; /**< output tensors config */
   GList *apply; /**< Select the tensors to apply transformation */
+  guint transpose_rank;  /**< transpose rank's limit */
```

Comment:
This variable is useless (not updates and kept constant). You may keep using ```NNS_TENSOR_TRANSPOSE_RANK_LIMIT``` everywhere in the .c file.

Reply:
I removed useless variable.
Thank you 👍